pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md

```diff
@@ -36,7 +36,7 @@ After you finish the installation, see [Spark Operator custom resource documenta
 
 -   Using CLI
 
-    See [Application Deployment](../../application-deployment#deploy-an-application-with-a-custom-configuration) for details.
+    See [Application Deployment](../../application-deployment#deploy-an-application-with-a-custom-configuration-using-the-cli) for details.
```

Comment:
I mean, I don't need to make a suggestion here, but we shouldn't change this file. This file can get changed in the `main` branch if it needs to get updated (`develop` branch should just be for `/2.2/` version changes)

```suggestion
    See [Application Deployment](../../application-deployment#deploy-an-application-with-a-custom-configuration) for details.
```

Reply:
Moved to this PR: https://github.com/mesosphere/dcos-docs-site/pull/4039

Removed in this commit: 971bbfefe97ce10cc19b89d9b726724a20f160bc

Reply:
Solid - thanks @cbuto !